### ISSUE TITLE
Content Tagging / Categories

### ISSUE THREAD

**Issue by Mike-E-angelo** [StarbeamOne/home#9]
We had an issuance this morning:
https://alpha.starbeam.one/market/products/101b389f-b096-498a-7000-08dab07b8cca

Which looks AI but is actually human/traditional.

Ideally, we want an effective way to both label and display such characteristics.

**Comment by Mike-E-angelo** [comment]
The other obvious component to this is searching by tags/categories.

**Comment by greenbrook** [comment]
Do you have any concrete tags in mind? I mean, are you interested in descriptive tags like "portrait", "landscape", "architecture", "fantasy". Or only in the necessary like "human-made"?

**Comment by Mike-E-angelo** [comment]
Well since we've done this we have the Traditional Market which is the "human-made" (or `hand-made` tag).  Technically speaking that is all the Traditional Market is, is a content/tag filter.

I do not have a list yet but it would be nice to get one started.  We have several themes:

Animals, dogs, cats, pugs, pizza. :)

**Comment by greenbrook** [comment]
+ cups, castles, girls :D

**Comment by Mike-E-angelo** [comment]
✅
- caves, torches, 8-bit :)

**Comment by greenbrook** [comment]
So, are you planning to add a tagline similar to DeviantArt?

<img width="889" alt="Знімок екрана 2023-03-07 о 19 17 30" src="https://user-images.githubusercontent.com/125400168/223498949-e49ab489-9346-4f66-a390-26ff39ce4f87.png">

**Comment by Mike-E-angelo** [comment]
Yes, this is a good idea :D  Please do continue providing valuable feedback/ideas/analysis 🙏👍

**Comment by Mike-E-angelo** [comment]
Related: #124
